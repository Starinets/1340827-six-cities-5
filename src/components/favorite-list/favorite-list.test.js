import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import FavoriteList from './favorite-list';
import {
  emptyFunction,
  City,
  offers
} from '../../test-data/test-data';

const favoriteList = [
  {
    city: City.PARIS,
    favorites: offers
  },
  {
    city: City.DUSSELDORF,
    favorites: offers
  },
];

it(`Component rendered correctly -> FavoriteList`, () => {
  const renderer = new ShallowRenderer();
  const tree = renderer.render(
      <FavoriteList
        favoriteList={ favoriteList }
        onCityClick={ emptyFunction }
      />
  );
  expect(tree).toMatchSnapshot();
});
