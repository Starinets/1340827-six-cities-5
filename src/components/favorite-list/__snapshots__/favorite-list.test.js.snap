// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component rendered correctly -> FavoriteList 1`] = `
<section
  className="favorites"
>
  <h1
    className="favorites__title"
  >
    Saved listing
  </h1>
  <ul
    className="favorites__list"
  >
    <FavoriteItems
      favoritesByCity={
        Object {
          "city": "Paris",
          "favorites": undefined,
        }
      }
      onCityClick={[Function]}
    />
    <FavoriteItems
      favoritesByCity={
        Object {
          "city": "Dusseldorf",
          "favorites": undefined,
        }
      }
      onCityClick={[Function]}
    />
  </ul>
</section>
`;
